core/state_processor_rip7560.go

```diff
@@ -352,7 +352,13 @@ func ApplyRip7560ValidationPhases(
 	header *types.Header,
 	tx *types.Transaction,
 	cfg vm.Config,
+	estimateFlag ...bool,
```

Comment:
I'm not sure, but how do you think about making the name of this flag `allowSigFail`, as in at `abiDecodeAcceptAccount`?